src/app/config/app.config.ts

```diff
@@ -14,7 +14,24 @@ import {ErrorHelper} from '../utilities/error-helper';
 @Injectable()
 export class AppConfig {
 
+  public static DEFAULT_API_URL = 'https://transmart-dev.thehyve.net';
+  public static DEFAULT_API_VERSION = 'v2';
+  public static DEFAULT_APP_URL = 'http://localhost:4200';
+  public static DEFAULT_APP_VERSION = '0.0.1-SNAPSHOT';
```

Comment:
Maybe the default app version should be something like 'no version specified'

Reply:
good idea, will change